contrib/templates/default-scala/README.md

```diff
@@ -0,0 +1,12 @@
+# default-scala
+
+This template helps you create Scala projects with Databricks Asset Bundles. It uses sbt to compile and package Scala files, and can be used with Databricks Connect for local development.
+
+It supports two compute types: standard and serverless compute.
```

Comment:
This says "standard" again; should it not say "standard clusters"? Or are we going with "standard" as a name in the product when we mean non-serverless UC compute?